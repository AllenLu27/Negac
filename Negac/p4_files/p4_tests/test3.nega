void fn_output(int a, int b){
	output a;
	output b;
	output "tesd3";
}

int main(){
     fn_output(1, 2);
     return(0);
}

