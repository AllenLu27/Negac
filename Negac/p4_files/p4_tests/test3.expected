[BEGIN GLOBALS]
str_0 "tesd3"
[END GLOBALS]
[BEGIN fn_output LOCALS]
a (formal)
b (formal)
[END fn_output LOCALS]
enter fn_output
getin 1 a
getin 2 b
OUTPUT a
OUTPUT b
OUTPUT str_0
lbl_0: leave fn_output
[BEGIN main LOCALS]
[END main LOCALS]
enter main
setin 1 1
setin 2 2
call fn_output
setout 1 0
goto lbl_1
lbl_1: EXIT
